### PR TITLE
DOCSP-25843 Release Notes for Atlas CLI v1.2.0 and v1.2.1

### PR DESCRIPTION
**Pull request by JuliaMongo**
[DOCSP-25843](https://jira.mongodb.org/browse/DOCSP-25843)

Staging: [Changelog](https://docs-mongodbcom-staging.corp.mongodb.com/atlas-cli/docsworker-xlarge/DOCSP-25843/atlas-cli-changelog/)

[The latest build](https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=634d6e6cf648ddbe1505ad74) is clean.

### CODE REVIEW

File: source/atlas-cli-changelog.txt [hunk 1]
```diff
@@ -12,6 +12,38 @@
...
+  prevented printing the command's results.
+- Fixes an issue with the :ref:`atlas-privateEndpoints-azure-interfaces-describe`
+  command where the output didn't display correctly.
+- Adds the :ref:`atlas-processes-describe` command.
+- Drops support for Debian 9.
+- Fixes a bug where inviting users issued an error.
+- Improves help text for the :ref:`atlas-backup-snapshot-create` command.
+- Fixes a bug where login information wasn't stored at early exit while
+  asking about project ID and org ID.
+- Adds a command to open the config file in your proffered editor.
```
Comment (gssbzn) [inline source/atlas-cli-changelog.txt:42]:
```suggestion
- Adds a command to open the config file in your preferred editor.
```

**Comment by gssbzn** [comment]
>It is not clear to me why the changelog file has release notes for 1.7.1 and then 1.2.0/1.2.1. 

The CLI follows [semver](https://semver.org/), <major>.<minor>.<patch>, 
We should bump <minor> when introducing features in a backward compatible way, ie new commands/flags
We should bump <patch> when only doing bug fixes, ie no new features
We should bump <major> when a breaking change

**Review by gssbzn — APPROVED** [review]
LGTM thanks for the fix

File: source/atlas-cli-changelog.txt [hunk 2]
```diff
@@ -12,6 +12,39 @@
...
 
+.. _atlas-cli-1.2.1:
+
+{+atlas-cli+} 1.2.1
+-------------------
+
+Released 14 October 2022
+
+- Fixes internal release tooling.
+- Makes ARM binaries available for installation with ``apt yum`` and ``dnf``.
```
Comment (sarahsimpers) [inline source/atlas-cli-changelog.txt:23]: Apt and Yum should be separate
```suggestion
- Makes ARM binaries available for installation with ``apt``, ``yum``, and ``dnf``.
```

**Pull request merged**